### PR TITLE
ECR + EKS + certificate

### PR DESCRIPTION
**Pull request by gjovani-vega**

### CODE REVIEW

File: infra/certificate.tf [hunk 1]
```diff
@@ -0,0 +1,21 @@
+data "aws_ssm_parameter" "kurs_omega_domain" {
```
Comment (Igor-Knezevic-VegaIT) [inline infra/certificate.tf:1]: You don't need **aws_ssm_parameter** at all.
Please use your domain as terraform variable.

File: infra/eks.tf [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  version = "~> 20.0"
+
+  cluster_name    = "eks-cluster-gjovani"
+  cluster_version = "1.29"
+
+  vpc_id     = module.vpc.vpc_id
+  subnet_ids = module.vpc.private_subnets
+
+  cluster_endpoint_public_access       = true
+  cluster_endpoint_public_access_cidrs = ["${data.aws_ssm_parameter.office_ip.value}/32"]
```
Comment (Igor-Knezevic-VegaIT) [inline infra/eks.tf:16]: You don't need **aws_ssm_parameter** at all.
Please use your IP as terraform variable.

File: infra/iam.tf [hunk 3]
```diff
@@ -42,10 +42,35 @@ module "iam_policy" {
           "ecr:CompleteLayerUpload",
           "ecr:PutImage"
         ],
-        "Resource": "*"
+        "Resource": "${module.ecr.repository_arn}"
```
Comment (Igor-Knezevic-VegaIT) [inline infra/iam.tf:45]: Check docs, this is not enough in order to work, you need auth token as well.

**Comment by gjovani-vega** [comment]
@Igor-Knezevic-VegaIT can you please check?

File: infra/iam.tf (hunk 3 above)
Comment (Igor-Knezevic-VegaIT) [reply infra/iam.tf:45]: You have auth token defined twice, check docs for exact permissions needed.

https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-push-iam.html

**Pull request merged**